src/library/dlgtrackinfo.cpp

```diff
@@ -744,7 +748,10 @@ mixxx::UpdateResult DlgTrackInfo::updateKeyText() {
 }
 
 void DlgTrackInfo::displayKeyText() {
-    const QString keyText = m_trackRecord.getMetadata().getTrackInfo().getKeyText();
+    const QString keyText =
+            KeyUtils::tryConvertKeyString(
+                    m_trackRecord.getMetadata().getTrackInfo().getKeyText(),
+                    m_notation);
```

Comment:
The issue is probably caused of hard to used code. We have here a hint:
https://github.com/mixxxdj/mixxx/blob/9817e5eab23ce907f8e49bfdb0050fef9549257d/src/track/track.cpp#L1453
but not here: 
https://github.com/mixxxdj/mixxx/blob/24a84b9d64607bb2b2cdcfc270bf9cb885074f3a/src/track/trackinfo.h#L35

The later seems to ba a container fr the key text stored in track metadata. Do we want to rename (again) it or at least add a comment? 

I think here we want exactly what found in the metadata, right?  
https://github.com/mixxxdj/mixxx/blob/9817e5eab23ce907f8e49bfdb0050fef9549257d/src/library/browse/browsethread.cpp#L239